src/app/modules/library/learning-resources/learning-resources.component.html

```diff
@@ -0,0 +1,22 @@
+<div>
+    <mat-toolbar class="c-filters">
+        <div *ngFor="let data of filtersData">
+            <mat-form-field style="width: 250px;" class="example-chip-list">
```

Comment:
Why are you hardcoding the width?

Reply:
Default the  size of the mat-form-field is very small as per the design i changed